### PR TITLE
let bundler fetch http_parser.rb submodules

### PR DESCRIPTION
**Pull request by ghost**
Motavation for this change:

The http_parser.rb gem uses git submodules and fetches them during the build. In situations where the dependencies for mastodon are fetched upfront, and a network connection is intentionally unavailable during the build, this will make the http_parser.rb gem build fail.
This is the case when packaging Mastodon with the Nix package manager.
Adding the submodules attribute makes bundler fetch the submodules upfront instead of them being fetched during the build process.

### CODE REVIEW

**Comment by Gargron** [comment]
No, it is related:

    The git source https://github.com/tmm1/http_parser.rb is not yet checked out. Please run `bundle install` before trying to start your application

**Comment by petabyteboy** [comment]
It was failing with a different, unrelated error first (fuubar 2.4.0 could not be fetched). Now I rebased on master and it gives this different error. I will look into it later.

File: Gemfile.lock [hunk 1]
```diff
@@ -22,6 +22,7 @@ GIT
   remote: https://github.com/tmm1/http_parser.rb
   revision: 54b17ba8c7d8d20a16dfc65d1775241833219cf2
   ref: 54b17ba8c7d8d20a16dfc65d1775241833219cf2
+	submodules: true
```
Comment (Shleeble) [inline Gemfile.lock]: space here

Comment (petabyteboy) [reply Gemfile.lock]: Thanks for catching that mistake!

**Comment by petabyteboy** [comment]
The build failure was caused by the whitespace error @Shleeble found, now the build passes.

**Review by Gargron — APPROVED** [review]

**Pull request merged**